src/deps.ts

```diff
@@ -256,10 +268,10 @@ export function getMongoDBClientEncryption():
     // Cannot be moved to helper utility function, bundlers search and replace the actual require call
     // in a way that makes this line throw at bundle time, not runtime, catching here will make bundling succeed
     mongodbClientEncryption = require('mongodb-client-encryption');
-  } catch (cause) {
+  } catch (error) {
     const kModuleError = new MongoMissingDependencyError(
       'Optional module `mongodb-client-encryption` not found. Please install it to use auto encryption or ClientEncryption.',
-      { cause }
+      { cause: error, dependencyName: 'mongodb-client-encryption' }
```

Comment:
Why the refactor away from shorthand properties?

Reply:
Do you mean `cause -> error`?  Primarily for consistency with the rest of the file, since they all catch errors named `error`.

Reply:
Sure, I don't mind either way, I prefer naming it cause for the shorthand but just my 🧅